api/handlers_spaces.go

```diff
@@ -15,12 +15,60 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// getTimeDefault returns time range from beginning of month to day-of-month now
+func getTimeDefault() (string, string, bool) {
+	log.Debug("no start or end time given on API input, assigning defaults")
```

Comment:
this doesn't make sense in this context actually - move the log message out to where it's called i think

Reply:
moved